src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -694,6 +694,8 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
     std::string device_id = GetConfig(ov::device::id.name(), options);
 
     auto iter = device_map.find(device_id);
+    if (iter == device_map.end())
+        iter = device_map.find(std::to_string(cldnn::debug_configuration::device_id));
```

Comment:
I think it's problem about priority of options because `-d GPU` implies `-d GPU.0` and it cause device_id conflict. Should I prioritize DEBUG_FLAG more than command line option?

Reply:
yes, debug_config should override cmdline option as it is a "more explicit" request.

Reply:
If we do not concern conflicts, default value of `debug_configuration::device_id` 0 looks good but we cannot determine whether DEBUG_CONFIG used or not. I'll update it to use -1 as default value.

Reply:
Done.

Reply:
Wiki page https://github.com/openvinotoolkit/openvino/wiki/GPUPluginDebugUtils#select-a-device-to-run